src/models/userModel.js

```diff
@@ -0,0 +1,32 @@
+const admin = require('../config/firebase');
+const firestore = admin.firestore();
+
+const createUserRecord = async (adminUid, userUid) => {
+  try {
+    await firestore.collection('adminUsers').doc(userUid).set({
```

Comment:
I don't think we've talked about `adminUsers` as a collection, or the `createdBy` field on a user. I would either like to see documentation identifying this collection, or even better we should just have one "users" collection for all our user objects and use the `isAdmin` field.

I don't think we should need to know what admin created the user. What if a user signs up on their own? We should be able to re-use the same code when a user signs up vs and admin creating the user.

I personally don't think we should have admins creating users at all. It's just extra work for Olga, users should sign up on their own. Whichever way you decide, make sure you talk with her.

Reply:
The team is on board with deleting create user functionality, so I will delete this part as it is related to viewing all the students that teachers created.